packages/react-app/.env.example

```
@@ -1,2 +1,3 @@
 VITE_OAUTH_CLIENT_ID=changeme
 VITE_GOOGLE_OAUTH_URI=https://www.googleapis.com/oauth2/v1/
+VITE_LUCKDART_URI=http://localhost:3000
```

Since the name can change in the future:
```suggestion
VITE_FRONTEND_URI=http://localhost:3000
```